src/test/java/games/strategy/triplea/delegate/AirThatCantLandUtilTest.java

```diff
@@ -26,19 +26,19 @@
 import games.strategy.triplea.xml.TestMapGameData;
 
 public class AirThatCantLandUtilTest {
-  private GameData m_data;
-  private PlayerID m_americans;
-  private UnitType m_fighter;
+  private GameData gameData;
+  private PlayerID americansPlayer;
+  private UnitType fighterType;
```

Comment:
I'd prefer to have this renaming in a separate PR in the future but we can leave it for now.